Final status:
Process still alive after running test suite
stdout and stderr:
DEBUG:root:[*] running method core_negotiate_tlv_encryption
DEBUG:root:[*] Negotiating TLV encryption
DEBUG:root:[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100c9028e106612523a66685fd6f49b484aac38d5b61077af045e52adbc54fa94e89477b25a6a2be5bf08c6e8e8bf81678e841514cab511f4fc23c64fb084418310dec9a05368f38631a533329bcf4a1a63dc75ec1266f4b42e7b697527a074676e628df9aa9f71b418e957199d0b9a50f6f42e6b743a44846a8d5ae022e2d76f9c55a1d832c237c482dd34e46204557438399fd89c798325d4b94e480d6892b788c14635a5cc727f0c0f5d73b7dc61c1f529a04024561286e18145f900d9a63db2ba29b8718521fce68ff487cfa54ca29b7a76ab24f40b76668ecc298231318e1b45e62e024a1b3d82f35adffc6f643b63e6fbc37baeaeb25944075e2bdeb0ca330203010001
DEBUG:root:[*] AES key: 0x3e4f79b729e60e1242610a3675848a41f289d5683f929ef32385d48bb336e964
DEBUG:root:[*] Encrypted AES key: 0x291485597a77d63a0f0d243c9b393c852436880163cbd9d48923ce2a0a7ea9c2055307cb99d9d39324eecc7066eca46632a1bd0c7c12e2bb7a48d2e1a0c3d80e22a52799f85b6d205828ecbf3faafb25ff16b6d4f52b8dafe6405e485473092f76c0287cdffdbd58047e5cceed3764e4279291d94d2990e31853721a3d7393c70df0b5184ff7e4ad328892e78f58ca60288ecb6a8562f06a6e343ea638950434b35c88703e7c067e1f20a9ef19c6a1d541c757dc67090c24112f11e7279fdb3a22f73384ad10f1ff8f1c8ace066422f67a2e7b12091ef2509bab66d1ce07bf205ee8cf95ab96dfe9dba10f00464113f5f190e06579bbf74c9f4668554c69ba48
DEBUG:root:[*] TLV encryption sorted
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_session_guid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_enumextcmd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_loadlib
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_sysinfo
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_set_uuid
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_interfaces
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_net_config_get_routes
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_separator
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-gebintqc'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 1 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 2 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 3 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 4 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 4
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 5 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 5
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '/etc/master.passwd'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 6 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 6
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 7 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 7
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 8 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 8
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 9 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 9
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 10 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 10
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 11 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_eof
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 11
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 12 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 12
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_file_move
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_file
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 13 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 13
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 14 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 14
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_ls
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo EEOTJTLI']
DEBUG:root:[*] added process id: 3330, handle: 1
DEBUG:root:[*] added channel id: 15 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[-] method core_channel_read resulted in error: #1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 1
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 16 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 16
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
DEBUG:root:[*] added process id: 3334, handle: 2
DEBUG:root:[*] added channel id: 17 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 2
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[-] method stdapi_fs_stat resulted in an error
DEBUG:root:Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_mkdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_open
DEBUG:root:[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
DEBUG:root:[*] added channel id: 18 type: MeterpreterFile
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_write
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_close
DEBUG:root:[*] closed and removed channel id: 18
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_execute
DEBUG:root:[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
DEBUG:root:[*] added process id: 3340, handle: 3
DEBUG:root:[*] added channel id: 19 type: MeterpreterProcess
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method core_channel_read
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_process_close
DEBUG:root:[*] closed and removed process handle: 3
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_stat
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_delete_dir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_sys_config_getenv
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_chdir
DEBUG:root:[*] sending response packet
DEBUG:root:[*] running method stdapi_fs_getwd
DEBUG:root:[*] sending response packet
